stages/org.osbuild.chattr

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python3
+"""
+Runs `chattr` to set file/directory attributes.
+"""
+
+import os
+import subprocess
+import sys
+from typing import Dict
+from urllib.parse import ParseResult, urlparse
+
+import osbuild.api
+
+SCHEMA_2 = r"""
+"options": {
+  "additionalProperties": false,
+  "required": ["path", "immutable"],
+  "properties": {
+    "immutable": {
+      "type": "boolean",
+      "description": "Make the file/directory immutable",
+      "default": true
+    },
+    "path": {
+      "oneOf": [
+        {
+          "type": "string",
+          "description": "The destination, if a mount",
+          "pattern": "^mount:\/\/[^\/]+\/"
+        },
+        {
+          "type": "string",
+          "description": "The destination, if a tree",
+          "pattern": "^tree:\/\/\/"
+        }
+      ]
+    }
+  }
```

Comment:
Let's follow the chmod and chown stage schemas here:
```json
"items": {
  "tree:path/to/item": {
    "immutable": true
  },
  "mount://mountpoint/": {
    "immutable": true
  }
}
```

Good for consistency and makes it possible to apply different attribute changes to multiple items.

Reply:
done in latest upload